q5.sql

```diff
@@ -0,0 +1,25 @@
+with bus_stops_accessible as (
+     select
+        stop_name,
+        wheelchair_boarding,
+        the_geom
+    from septa_bus_stops),
+	
+	bus_neighbors as (
+	 select ph.name, ph.geom, bs.wheelchair_boarding,
+	        ST_AREA(ph.geom) as area_size
+     from phl_neighborhoods as ph
+	 join septa_bus_stops as bs
+	 on st_Within(ST_Transform(ph.geom,4326), ST_Transform(bs.the_geom,4326))),
```

Comment:
It looks like your arguments to `ST_Within` are backwards. The contained geometry should be first, and the containing geometry second.